tools/crm_mon_print.c

```diff
@@ -507,7 +424,23 @@ print_rsc_history_start(mon_state_t *state, pe_working_set_t *data_set,
 
         /* Print last failure time if any */
         if (last_failure > 0) {
-            print_nvpair(state, CRM_LAST_FAILURE_PREFIX, NULL, NULL, last_failure);
+            switch (state->output_format) {
+                case mon_output_console:
+                case mon_output_plain:
+                    print_as(state->output_format, " %s",
+                             pcmk_format_named_time(CRM_LAST_FAILURE_PREFIX, last_failure));
```

Comment:
I am positive I already did this.  I wonder if it somehow got lost in a rebase.  Oh well, guess I'll go do it again.

Reply:
I corrected several more of these.  For pcmk_format_named_time, there's nowhere it's being called where it is not freed, so this must have gotten fixed in an intermediate commit.  I am having some trouble tracking down which one it was.

Reply:
Okay, I went through and made sure each patch where one of these functions is called also has a free, so they should all be right now.  If something isn't getting freed in the patch where it's introduced, point it out and I'll take care of it.  Note there's still a patch that adds frees of time_t_string calls but that's because those calls don't come from this PR.